views/campaign-signup.ejs

```diff
@@ -0,0 +1,4 @@
+<!-- Re-using Partner Microsite styles -->
+<div class="partner-page">
```

Comment:
Same. I just wasn't sure if the campaign landing page would always be the same or if it would eventually deviate from the partner page designs.